tests/integration/targets/mso_schema_template_bd/tasks/main.yml

```diff
@@ -683,14 +712,7 @@
     - query_validate.msg is match ("MSO Error 400{{':'}} Bad Request{{':'}} Patch Failed, Received{{':'}} Template 'Template5', BD 'ansible_test_5' {{':'}} ARP Flooding has to be disabled if L2 Stretch enabled and BUM traffic disabled exception while trying to update schema")
   when:
   - version.current.version is version('3.3', '>=')
-  - version.current.version is version('3.6', '<')
-
-- name: Verify query_validate for a version that's after 3.3
-  assert:
-    that:
-    - query_validate is not changed
-    - query_validate.msg is match ("MSO Error 400{{':'}} Bad Request{{':'}} Patch Failed, Received{{':'}} BD 'ansible_test_5' l2UnknownUnicast cannot be flood when intersiteBumTrafficAllow is off exception while trying to update schema")
-  when: version.current.version is version('3.6', '>=')
+  - version.current.version is version('3.7', '<')
```

Comment:
mso_schema_validate not needed after 4.0 because validation is done upon request, validation result is seen in task - name: Verify nm_change_bd_5_3 for a version that's after 4.0